### PR TITLE
Misc fixes

### PR DESCRIPTION
**Pull request by tamird**
@petermattis @tschottdorf @BramGruneir

### CODE REVIEW

**Comment by tschottdorf** [comment]
Flush needs to be errchecked from the looks of it.

**Comment by tamird** [comment]
Yeah will come back to this when https://github.com/jteeuwen/go-bindata/pull/91 and https://github.com/github/linguist/pull/2426 are merged.

File: server/cli/cli.go [hunk 1]
```diff
@@ -43,6 +43,9 @@ Output build version information.
 		fmt.Fprintf(w, "Build Time:  %s\n", info.Time)
 		fmt.Fprintf(w, "Build Deps:\n\t%s\n",
 			strings.Replace(strings.Replace(info.Deps, " ", "\n\t", -1), ":", "\t", -1))
+		if err := w.Flush(); err != nil {
+			fmt.Println(err)
```
Comment (tschottdorf) [inline server/cli/cli.go]: nit:

either `_ = w.Flush()`
or
```
if err := w.Flush(); err != nil {
    panic(err.Error())
}
```

unless there's a way to return an honest exit code to the shell.

Comment (tamird) [inline server/cli/cli.go]: I would expect this error to be something to do with malformed tabular input, not a real failure to flush the buffer. why not just print it?

Comment (tschottdorf) [inline server/cli/cli.go]: it's an unknown error for all we know, so either we handle it or we don't. Seems predestined for a panic if you ask me.

**Comment by tschottdorf** [comment]
LGTM mod Fatalf or something like that for the Flush error.

**Pull request merged**